src/components/Dashboard/Form/Applicants/Applicant.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../../scss/_mixins.scss';
+.Applicant{
+  @include flex(wrap);
+  width: 650px ;
+}
+button.developedDiv {
```

Comment:
use only class ;) don't mix that with tag, css modules give us the unique classes anyway

Reply:
done